migrations/versions/890_agreement.py

```diff
@@ -0,0 +1,44 @@
+"""signed agreement
+
+Revision ID: dbdf582a1fa3
+Revises: 880
+Create Date: 2017-01-17 13:31:31.855390
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '890'
+down_revision = '880'
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+def upgrade():
+    ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('agreement',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('version', sa.String(), nullable=False),
+    sa.Column('url', sa.String(length=15), nullable=False),
```

Comment:
Postgres tip: Skip all lengths. There are no performance advantages to setting a length